spec.html

```diff
@@ -33172,19 +33166,19 @@
 
       <!-- es6num="24.2.2.1" -->
       <emu-clause id="sec-dataview-buffer-byteoffset-bytelength">
-        <h1>DataView (_buffer_, _byteOffset_, _byteLength_ )</h1>
-        <p>`DataView` called with arguments _buffer_, _byteOffset_, and _byteLength_ performs the following steps:</p>
+        <h1>DataView (_buffer_ [ , _byteOffset_, _byteLength_ ] )</h1>
```

Comment:
for the same clarity reason I mentioned on the ArrayBuffer constructor, I believe the brackets are good to tell they are optional.